products/bash/validate-releases.sh

```diff
@@ -0,0 +1,179 @@
+#!/bin/bash
+#******************************************************************************
+# Licensed Materials - Property of IBM
+# (c) Copyright IBM Corporation 2020. All Rights Reserved.
+#
+# Note to U.S. Government Users Restricted Rights:
+# Use, duplication or disclosure restricted by GSA ADP Schedule
+# Contract with IBM Corp.
+#******************************************************************************
+#
+# INSTRUCTIONS
+# ------------
+#
+# 1. Run the script, passing the Cloud Pak console address as argument:
+#       ./validate-releases.sh icp-console.<your-cluster-domain>
+#
+# 2. It will use 'admin' to login to the console, and prompt for the password;
+#    you can change the username or set the password in the environment:
+#       export CP_USERNAME=<username>
+#       export CP_PASSWORD=<password>
+#
+# 3. You can specify release names for each product using the environment variables:
+#       export ACE_DASHBOARD_RELEASE_NAME=<ace-dashboard-release-name>
+#       export ACE_DESIGNER_RELEASE_NAME=<ace-designer-release-name>
+#       export APIC_RELEASE_NAME=<apic-release-name>
+#       export ASSET_REPO_RELEASE_NAME=<asset-repo-release-name>
+#       export EVENT_STREAMS_RELEASE_NAME=<event-streams-release-name>
+#       export MQ_RELEASE_NAME=<mq-release-name>
+#       export TRACING_RELEASE_NAME=<tracing-release-name>
+#
+# 4. It will release all supported products by default;
```

Comment:
We know in the future we will want to release multiple instances of some products. I'm thinking we should future proof this script now rather than trying to keep consistency with release-products.sh. It would be much more general purpose if the input parameters was a list of release names rather than product names. I.e. when we change to install multiple instances of ace dashboard this script could be called with both release names and not need to be changed.